app/src/main/res/values/strings.xml

```diff
@@ -5,8 +5,7 @@
     <string name="destination_location_label">Destination Location</string>
     <string name="longitude_label">Longitude</string>
     <string name="latitude_label">Latitude</string>
-    <string name="netguru_offices_label">Netguru Offices</string>
-    <string name="gdansk_office_label">Gdansk Office</string>
+    <string name="find_atm_label">Find ATM</string>
     <string name="latitude_error">Latitude value should be between -90 an 90</string>
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/ar-localizer-view-android/blob/6e7a43f049fda9487b1419ac862329af39e581fa/app/src/main/res/values/strings.xml#L9"></span>The resource <code>R.string.latitude_error</code> appears to be unused</td>
    </tr>
  </tbody>
</table>